docs/userguide/scheduling/override-policy.md

```diff
@@ -473,5 +474,58 @@ data:
   example: 2
 ```
 
+### FieldOverrider
```

Comment:
I think it would be great to add the `FieldOverrider` ahead of `PlaintextOverrider`, just to keep consistent with the apply order.

Reply:
Thanks for reminding. I changed it.